app/views/admin/tag_sets/new.html.erb

```diff
@@ -0,0 +1,26 @@
+
+<div class="page-header"><h1>New Tag Set</h1></div>
+
+<%= form_for @tag_set, url: admin_tag_sets_path do |f| %>
+  <%= render_error_messages(@tag_set) %>
```

Comment:
@yoldas and @dasunpubudumal isn't this where the errors from a failed save attempt appear?

Reply:
oh yeah apparently this is declared in `application_helper.rb` and adds a `div` with errors. All good!